### PR TITLE
[Draft]feat: update drf_yasg library for python 38

### PR DESCRIPTION
**Pull request by NeOneSoft**
### Description
This PR was created from [PR](https://github.com/openedx/credentials/pull/1502) in order to solve  conflicts with versions dependencies

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @NeOneSoft!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by natabene** [comment]
@NeOneSoft Thank you for your contribution. Is this ready for our review?

**Review by hurtstotouchfire — APPROVED** [review]

**Comment by hurtstotouchfire** [comment]
@oliviaruizknott Can you take a look and try to get this merged before merging any other python updates? We keep creating merge conflicts and I'd like to just get this in.

**Review by oliviaruizknott — CHANGES_REQUESTED** [review]
Hi @NeOneSoft—

I know this PR has been through a lot. Thank you for your patience! It looks like this is failing a linting check. I believe it will be fixed if you run:
```
isort credentials/urls.py
```
—and then commit + push these automatically-applied changes.

Once that’s done and the tests pass, I believe we can merge this!

**Comment by NeOneSoft** [comment]
Hi @oliviaruizknott  and @hurtstotouchfire!!Thank you so much for all your support. I'll be glad to attend your comments. For now I already applied the suggested import sort changes hoping that this fix the lint error. Thanks again!!!!

**Comment by NeOneSoft** [comment]
Sorry, I noticed that a reformat issue came up but I'm wondering if the import sorted triggered this new failed. @oliviaruizknott

**Comment by oliviaruizknott** [comment]
Ah, another linter (`black`) is complaining about a new problem:

```
--- credentials/urls.py	2022-06-28 20:15:53.884620 +0000
+++ credentials/urls.py	2022-06-28 20:21:36.087851 +0000
@@ -35,24 +35,24 @@
 admin.autodiscover()
 admin.site.site_header = _("Credentials Administration")
 admin.site.site_title = admin.site.site_header

 schema_view = get_schema_view(
-   openapi.Info(
-      title="Credentials API",
-      default_version='v1',
-      description="Credentials API docs",
-   ),
-   public=False,
-   permission_classes=[permissions.AllowAny],
+    openapi.Info(
+        title="Credentials API",
+        default_version="v1",
+        description="Credentials API docs",
+    ),
+    public=False,
+    permission_classes=[permissions.AllowAny],
 )

 urlpatterns = oauth2_urlpatterns + [
     re_path(r"^admin/", admin.site.urls),
     re_path(r"^api/", include(("credentials.apps.api.urls", "api"), namespace="api")),
     re_path(r"^api-auth/", include((oauth2_urlpatterns, "rest_framework"), namespace="rest_framework")),
-    re_path(r"^api-docs/$", schema_view.with_ui('swagger', cache_timeout=0), name="api_docs"),
+    re_path(r"^api-docs/$", schema_view.with_ui("swagger", cache_timeout=0), name="api_docs"),
     re_path(r"^auto_auth/$", core_views.AutoAuth.as_view(), name="auto_auth"),
     re_path(r"^credentials/", include(("credentials.apps.credentials.urls", "credentials"), namespace="credentials")),
     re_path(r"^health/$", core_views.health, name="health"),
     re_path(
         r"^management/", include(("credentials.apps.edx_django_extensions.urls", "management"), namespace="management")
would reformat credentials/urls.py

All done! ✨ 🍰 ✨
1 file would be reformatted, 146 files would be left unchanged.
```

You can fix this by running `black .`. You may need to be in the credentials shell for this to work. Otherwise, you can make the above changes manually.

File: credentials/urls.py [hunk 1]
```diff
@@ -34,11 +36,21 @@
...
+   public=False,
+   permission_classes=[permissions.AllowAny],
+)
+
 urlpatterns = oauth2_urlpatterns + [
     re_path(r"^admin/", admin.site.urls),
     re_path(r"^api/", include(("credentials.apps.api.urls", "api"), namespace="api")),
     re_path(r"^api-auth/", include((oauth2_urlpatterns, "rest_framework"), namespace="rest_framework")),
-    re_path(r"^api-docs/", get_swagger_view(title="Credentials API"), name="api_docs"),
+    re_path(r"^api-docs/$", schema_view.with_ui('swagger', cache_timeout=0), name="api_docs"),
```
Comment (oliviaruizknott) [inline credentials/urls.py:53]:
```suggestion
    re_path(r"^api-docs/$", schema_view.with_ui("swagger", cache_timeout=0), name="api_docs"),
```

File: credentials/urls.py [hunk 2]
```diff
@@ -34,11 +36,21 @@
 admin.site.site_header = _("Credentials Administration")
 admin.site.site_title = admin.site.site_header
 
+schema_view = get_schema_view(
+   openapi.Info(
+      title="Credentials API",
+      default_version='v1',
+      description="Credentials API docs",
+   ),
+   public=False,
```
Comment (oliviaruizknott) [inline credentials/urls.py:45]: Needs 4 spaces, and then another 4 spaces of indentation:

```suggestion
    openapi.Info(
        title="Credentials API",
        default_version="v1",
        description="Credentials API docs",
    ),
    public=False,
    permission_classes=[permissions.AllowAny],
```

**Comment by NeOneSoft** [comment]
Thank you so much @oliviaruizknott !! I already applied the suggested changes and I will check if that works as expected!!.

**Review by oliviaruizknott — APPROVED** [review]

**Pull request merged**